product/carbon-home/conf/identity/domain-config.yaml

```diff
@@ -139,4 +139,4 @@ domains:
         databaseType: "MySQL"
         hashAlgorithm: "SHA256"
         keyLength: 256
-        iterationCount: 4096
+        iterationCount: 4096
```

Comment:
Keep the newline.

Reply:
Fixed with https://github.com/wso2/product-is/pull/1371